snappi_ixnetwork/trafficitem.py

```diff
@@ -907,21 +915,34 @@ def _configure_stack_fields(self, ixn_fields, snappi_header, stacks):
                 ind = field_names.index(val)
             except Exception:
                 continue
-            if (
-                field_map.get("convert_int_to_hex") is not None
-                and field in field_map["convert_int_to_hex"]
-            ):
-                format_type = "hex"
+            format_type = field_map.get("convert_int_to_hex", [])
+            format_type = "hex" if field in format_type else None
+            auto_to_def = field_map.get("auto_to_default", [])
```

Comment:
I am ok with current implementation. I am just thinking if we somehow got ```auto``` flag present from snappi internal object. Then we can remove ```auto_to_default``` map. And reduced recursive manual effort